src/aof.c

```diff
@@ -1216,7 +1216,7 @@ int aofCreatePipes(void) {
 
     if (pipe(fds) == -1) goto error; /* parent -> children data. */
     if (pipe(fds+2) == -1) goto error; /* children -> parent ack. */
-    if (pipe(fds+4) == -1) goto error; /* children -> parent ack. */
+    if (pipe(fds+4) == -1) goto error; /* parent -> children ack. */
```

Comment:
This comment was fixed and finally removed altogether with multipart AOF: 87789fae0b767e47fef78ee994434554f2bf2f31